app/src/main/dist/config/crate.yml

```diff
@@ -58,6 +58,74 @@
 # features of the Enterprise Edition is governed by the terms and conditions of
 # your Enterprise or Subscription Agreement with Crate.io.
 
+############################ JMX Monitoring Plugin ############################
```

Comment:
nah change the `#` to `/` I reckon

it makes it clear it's a subsection under the enterprise section